posthog/decorators.py

```diff
@@ -15,6 +15,7 @@
 
 
 class CacheType(str, Enum):
+    FILTER = "Filter"
     TRENDS = "Trends"
```

Comment:
Changing constant from 'trends' to 'filters' because the caching mechanism works for all filtering. Keeping the old constant so that currently cached items still work